prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -247,6 +247,9 @@ size_t rxr_pkt_req_max_data_size(struct rxr_ep *ep, fi_addr_t addr, int pkt_type
 		return rxr_env.shm_max_medium_size;
 	}
 
+	if (OFI_UNLIKELY(!(peer->flags & RXR_PEER_HANDSHAKE_RECEIVED)))
+		return ep->mtu_size - rxr_pkt_req_max_header_size(pkt_type) - ep->core_addrlen;
+
```

Comment:
Let's fix rxr_pkt_req_max_header_size() add ep->core_addrlen to sizeof(struct rxr_req_opt_raw_addr_hdr)

Reply:
Also, since you are here, can you add ```assert(tx_entry->total_len == rxr_req_pkt_data_size(pkt_entry));``` in rxr_pkt_init_eager_msgrtm() and rxr_pkt_init_eager_tagrtm()

Reply:
Done.